apps/ledger-live-desktop/tests/models/AccountPage.ts

```diff
@@ -43,4 +43,17 @@ export class AccountPage {
     await this.stakeBannerButton.scrollIntoViewIfNeeded();
     await this.stakeBannerButton.click();
   }
+
+  async scrollToOperations() {
+    await this.page.locator("id=operation-list").scrollIntoViewIfNeeded();
+  }
+
+  /**
+   * Delete account from account itself
+   */
+  async deleteAccount() {
+    await this.page.locator("data-test-id=account-settings-button").click();
```

Comment:
Looks fine but why not declare these as reusable locators in the constructor?

Reply:
You're right, I forgot to rewrite this page object 👍